docker/Dockerfile.server

```diff
@@ -6,6 +6,11 @@
 # for why --platform
 FROM --platform=linux/amd64 python:3.9-bullseye
 
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get install -y sqlite3
+COPY install_sqlite3.sh .
+RUN bash install_sqlite3.sh
```

Comment:
Have you verified that this actually removes the message? IIUC, python will use a version of sqlite3 that comes bundled with the python distro, and that's what you'd need to upgrade. Though perhaps `make install` updates paths and such so that python will find the built one rather than its bundled one. So this might still work.

Reply:
To test, one can log into the produced docker image, run python, import sqlite3, and dump its version.